docs/installation.md

```diff
@@ -72,7 +72,8 @@ source ./staging/usr/local/bin/activate
 > `source ./staging/usr/local/bin/activate`
 
 ```shell-session
-python3 -m pip install --upgrade setuptools wheel cython==0.28.6
+python3 -m pip install --upgrade pip
```

Comment:
Can you remove the line about upgrading pip? It's not very related to install logdevice itself.

Reply:
I fixed by removing the commit. Is it correct?

Best regards,

Takayuki